### PR TITLE
let's improve it and expand the audience

### PR DESCRIPTION
**Pull request by jarektkaczyk**
And rename `laravel-json` to `json`, so your audience is userland, rather than just *artisans*.

### CODE REVIEW

**Comment by johannesschobel** [comment]
did you just re-write the whole package in order to improve the overall quality? :laughing: :+1:

**Comment by jarektkaczyk** [comment]
@johannesschobel yup

**Comment by johannesschobel** [comment]
@jarektkaczyk that is just awesome! you're a beast ^^

**Comment by jarektkaczyk** [comment]
@johannesschobel gimme more :fire:

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "eastwest/json",
-    "type": "library",
+    "type": "package",
```
Comment (svenluijten) [inline composer.json]: the `type` in `composer.json` should either remain at `library`, or be dropped entirely (it defaults to `library`). `package` is a made-up name that has no official support in Composer and possibly third-party plugins.

Ref: https://getcomposer.org/doc/04-schema.md#type

**Comment by jannejava** [comment]
@jarektkaczyk thanks for your PR. Not much left. ;-) I will look at it tomorrow. Good idea to move it outside the Laravel world.

**Comment by gocanto** [comment]
It looks good to me 🙃💁🏻‍♂️

File: composer.json (hunk 1 above)
Comment (jarektkaczyk) [reply composer.json]: Good catch.

Basically it does nothing and can be, indeed, omitted. 

For reference 
https://getcomposer.org/doc/04-schema.md#type

**Comment by paulredmond** [comment]
Solid, looks great 🥇

**Comment by jannejava** [comment]
@jarektkaczyk Good work!

**Comment by svenluijten** [comment]
As an aside, this will need to be re-submitted to Packagist, ~as well as (probably) have v2.0.0 tagged as it changes some naming/namspaces around~.

**Comment by jannejava** [comment]
@svenluijten Thanks for the reminder. (Finally got the auto-update on to work). Version 3.0.1 is now released on Packagist

**Pull request merged**